### DISCUSSION TITLE
What is this atom schema used for?

### DISCUSSION THREAD

**Discussion by Kate-Lyndegaard** [INSPIRE-MIF/helpdesk-validator#847] [Q&A] [answered]
Hi,

I would like to know what this atom schema is used for and where it comes from:

https://inspire-geoportal.ec.europa.eu/schemas/inspire/atom/1.0/atom.xsd

It seems the ATOM specification itself does not define any DTD or XML Schema and only requires the correct namespace to be used. So at least from the side of ATOM itself there is no requirement to specifically order elements in a certain way.

From https://www.rfc-editor.org/rfc/rfc4287.txt (page 5):

   Both kinds of Atom Documents are specified in terms of the XML
   Information Set, serialized as XML 1.0 [W3C.REC-xml-20040204] and
   identified with the "application/atom+xml" media type.  Atom
   Documents MUST be well-formed XML.  This specification does not
   define a DTD for Atom Documents, and hence does not require them to
   be valid (in the sense used by XML).

I could also not find any official XML schema for ATOM - it seems due to the above there is none.

The schema defines a mandatory element inspire_dls:spatial_dataset_identifier_code for an Atom entry. While this is required for the service feed, there is no mention that this should be present in a dataset feed, and the dataset feed examples from the guidelines don’t contain this either.

I would appreciate your feedback.

Kind regards,
Kate

**Comment by fabiovinci** [comment, marked as answer]
Dear @Kate-Lyndegaard,

the schema you mentioned was probably developed for the purpose of the Geoportal and/or for use in the very first version of the validator and it is not used by the current INSPIRE Reference Validator.

In any case, if you want to use it there is a specific XSD for the dataset feed (http://inspire-geoportal.ec.europa.eu/schemas/inspire/atom/1.0/atom_subfeed.xsd").

**Comment by Kate-Lyndegaard** [comment]
Dear @fabiovinci,

Thank you so much! This is exactly what I was looking for.

Kind regards,
Kate

**Comment by fabiovinci** [comment]
Please note that such schemas are not official and are not currently maintained, so they may not be fully in line with IR/TG.

**Comment by stempler** [comment]
Just as a note: These schemas are no longer available at the specified URLs, probably due to a change to the Geoportal.

**Reply by fabiovinci** [reply]
Dear @stempler,
thank you for the notification.
The Geoportal team is working on restoring all the content of the old Geoportal.
The schemas for ATOM, even if they are not official, will be restored soon.
